chapter06/mssw/exercise1.pl

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/perl
+use v5.10;
+use strict;
+use warnings;
+use Storable qw( nstore retrieve );
+use Data::Dumper;
+
+require '/home/codio/workspace/chapter06/mssw/file_reader.pm';
```

Comment:
only because relative was all relative to me executing them and not relative to the location of the file. It was annoying me :stuck_out_tongue: